build.sbt

```diff
@@ -1,82 +1,36 @@
-import play.sbt.routes.RoutesKeys
-import sbt.Keys.compile
 import sbt.Tests.{Group, SubProcess}
-import uk.gov.hmrc.DefaultBuildSettings.{addTestReportOption, defaultSettings, scalaSettings}
 
 val appName = "individuals-employments-api"
-val hmrc = "uk.gov.hmrc"
 
-TwirlKeys.templateImports := Seq.empty
-RoutesKeys.routesImport := Seq(
-  "uk.gov.hmrc.individualsemploymentsapi.Binders._"
-)
-
-lazy val playSettings: Seq[Setting[_]] = Seq(
-  routesImport ++= Seq(
-    "uk.gov.hmrc.domain._",
-    "uk.gov.hmrc.individualsemploymentsapi.domain._",
-    "uk.gov.hmrc.individualsemploymentsapi.Binders._"))
-
-lazy val appDependencies: Seq[ModuleID] = AppDependencies.compile ++ AppDependencies.test()
-lazy val plugins: Seq[Plugins] = Seq.empty
-def intTestFilter(name: String): Boolean = name startsWith "it"
-def unitFilter(name: String): Boolean = name startsWith "unit"
 def componentFilter(name: String): Boolean = name startsWith "component"
```

Comment:
How does it work now that you've removed test/it configurations?

Why did you leave out the component tests? It seems odd to combine it and unit tests and not component tests (I'd probably start with combining unit + component and then maybe it, since we don't have the distinction in other repos, so you can run all tests by typing `sbt test` so it's not surprising and you don't miss any tests)